src/main/java/com/example/demo/model/enums/jdbc/JdbcConnect.java

```diff
@@ -0,0 +1,20 @@
+package com.example.demo.model.enums.jdbc;
+
+
+public enum JdbcConnect {
```

Comment:
Configuration classes should not be represent as enum.
It is not a list of similar items.

Reply:
Done